src/global/utils/calc-benzo.js

```diff
@@ -0,0 +1,112 @@
+            if (eachBenzo.name === drugA) {
+                drugAResult = eachBenzo;
+                doseA = eachBenzo.diazvalue;
+                logger.debug(`[${PREFIX}] ${drugA} dose_a: ${doseA}`);
+            }
+            if (eachBenzo.name === drugB) {
+                drugBResult = eachBenzo;
+                doseB = eachBenzo.diazvalue;
+                logger.debug(`[${PREFIX}] ${drugB} dose_b: ${doseB}`);
+            }
+        }
+
+        const result = (dosage / doseA) * doseB;
+        let drugADosageText = '';
+        if (drugAResult.formatted_dose.Oral) {
+            console.log(`[${PREFIX}] ${drugA} is Oral`); // eslint-disable-line
+            drugADosageText = `\
+            ${drugAResult.formatted_dose.Oral.Light ? `Light: ${drugAResult.formatted_dose.Oral.Light}\n` : ''}\
+            ${drugAResult.formatted_dose.Oral.Low ? `Low: ${drugAResult.formatted_dose.Oral.Low}\n` : ''}\
+            ${drugAResult.formatted_dose.Oral.Common ? `Common: ${drugAResult.formatted_dose.Oral.Common}\n` : ''}\
+            ${drugAResult.formatted_dose.Oral.Heavy ? `Heavy: ${drugAResult.formatted_dose.Oral.Heavy}\n` : ''}\
+            ${drugAResult.formatted_dose.Oral.Strong ? `Strong: ${drugAResult.formatted_dose.Oral.Strong}\n` : ''}`;
+        } else if (drugAResult.formatted_dose.Light) {
+            console.log(`[${PREFIX}] ${drugA} is Light`); // eslint-disable-line
+            drugADosageText = `\
+            ${drugAResult.formatted_dose.Light.Light ? `Light: ${drugAResult.formatted_dose.Light.Light}\n` : ''}\
+            ${drugAResult.formatted_dose.Light.Low ? `Low: ${drugAResult.formatted_dose.Light.Low}\n` : ''}\
+            ${drugAResult.formatted_dose.Light.Common ? `Common: ${drugAResult.formatted_dose.Light.Common}\n` : ''}\
+            ${drugAResult.formatted_dose.Light.Heavy ? `Heavy: ${drugAResult.formatted_dose.Light.Heavy}\n` : ''}\
+            ${drugAResult.formatted_dose.Light.Strong ? `Strong: ${drugAResult.formatted_dose.Light.Strong}\n` : ''}`;
+        }
+
+        let drugBDosageText = '';
+        if (drugBResult.formatted_dose.Oral) {
+            console.log(`[${PREFIX}] ${drugA} is Oral`); // eslint-disable-line
+            drugBDosageText = `\
+            ${drugBResult.formatted_dose.Oral.Light ? `Light: ${drugBResult.formatted_dose.Oral.Light}\n` : ''}\
+            ${drugBResult.formatted_dose.Oral.Low ? `Low: ${drugBResult.formatted_dose.Oral.Low}\n` : ''}\
+            ${drugBResult.formatted_dose.Oral.Common ? `Common: ${drugBResult.formatted_dose.Oral.Common}\n` : ''}\
+            ${drugBResult.formatted_dose.Oral.Heavy ? `Heavy: ${drugBResult.formatted_dose.Oral.Heavy}\n` : ''}\
+            ${drugBResult.formatted_dose.Oral.Strong ? `Strong: ${drugBResult.formatted_dose.Oral.Strong}\n` : ''}`;
+        } else if (drugBResult.formatted_dose.Light) {
+            console.log(`[${PREFIX}] ${drugA} is Light`); // eslint-disable-line
+            drugBDosageText = `\
```

Comment:
'drugBDosageText' is assigned a value but never used.eslint[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)